skimage/measure/_find_contours_cy.pyx

```diff
@@ -25,6 +28,13 @@ def iterate_and_store(double[:, :] array,
     if array.shape[0] < 2 or array.shape[1] < 2:
         raise ValueError("Input array must be at least 2x2.")
 
+    cdef bint _nodata_enabled
+    cdef double _nd = 0
+
+    _nodata_enabled = nodata is not None
```

Comment:
Can you merge this line with the declaration above? Should be a little easier to read.

Reply:
Good catch. Fixed in 6bd1ae2.